rdrf/explorer/templates/explorer/base.html

```diff
@@ -14,7 +14,7 @@
         <script type="text/javascript" src="{% static 'js/vendor/underscore-1.13.1.min.js' %}"></script>
         
         <link rel="stylesheet" href="{% static 'css/jquery-1.13.0/themes/smoothness/jquery-ui.min.css' %}">
-        <link rel="stylesheet" href="{% static 'vendor/bootstrap-5.1.0/css/bootstrap.min.css' %}">
+        {% if LANGUAGE_BIDI %}<link rel="stylesheet" href="{% static 'vendor/bootstrap-5.1.0/css/bootstrap.rtl.min.css' %}">{% else %}<link rel="stylesheet" href="{% static 'vendor/bootstrap-5.1.0/css/bootstrap.min.css' %}">{% endif %}
```

Comment:
Might be nicer split over multiple lines. 
Could we minimise the duplication by just replacing the `bootstrap.min.css` with the required resource?
Is there a better DRY way to output the bootstrap stylesheet includes across the many templates we declare them for?

Reply:
Yeah, I'll give it a go. I think the rtl stylesheet is different than the ltr one, but will double check.

Reply:
Simplified the condition a bit. Other than that, we could create a custom template tag.

Reply:
Looks better 👍